<ui version="4.0" >
 <author>Aaron Racicot</author>
 <class>ExecuteToolGui</class>
 <widget class="QDialog" name="ExecuteToolGui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Tool Setup</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="variableBox" >
         <property name="title" >
          <string>Variables</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="executionBox" >
         <property name="title" >
          <string>Log/Status</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QProgressBar" name="progressBar" >
            <property name="value" >
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit" />
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4" >
      <attribute name="title" >
       <string>Tool Help</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QTextEdit" name="toolhelpEdit" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="widget" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>45</height>
      </size>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="execTool" >
        <property name="text" >
         <string>Execute Tool</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="cancelExec" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
